frontend/src/pages/index.ts

```diff
@@ -0,0 +1,9 @@
+/* Moments */
+export { MomentsView } from './moments/momentsView'
+export { MomentDetailView } from './moments/momentDetailView'
+/* Moments */
+export { ProfileView } from './profile/profileView'
+/* Moments */
+export { ProgressView } from './progress/progressView'
+/* Moments */
+export { ReflectionsView } from './reflections/reflectionsView'
```

Comment:
```suggestion
/* Moments */
export { MomentsView } from './moments/momentsView'
export { MomentDetailView } from './moments/momentDetailView'

/* Profile */
export { ProfileView } from './profile/profileView'

/* Progress */
export { ProgressView } from './progress/progressView'

/* Reflections */
export { ReflectionsView } from './reflections/reflectionsView'
```

Reply:
Did the Linter/Format added the spaces or did you manually?

Reply:
I added it manually for better readability. Not a fan?